Common/include/CConfig.hpp

```diff
@@ -9738,6 +9739,11 @@ class CConfig {
    */
   unsigned long GetEdgeColoringGroupSize(void) const { return edgeColorGroupSize; }
 
+  /*!
+   * \brief Check if the discrete adjoint is allowed to relax the coloring, that is, allow smaller edge color group sizes and allow more colors.
+   */
+  bool GetEdgeColoringRelaxDiscAdj(void) const { return edgeColoringRelaxDiscAdj; }
```

Comment:
```suggestion
  bool GetEdgeColoringRelaxDiscAdj() const { return edgeColoringRelaxDiscAdj; }
```

Reply:
Thanks for the remark. Is there general agreement in SU2 that we do it one way or the other? I saw that functions in `CConfig` have both variants, I decided to conform with the other edge coloring related function `GetEdgeColoringGroupSize(void)`.

Reply:
The formatting and clang tools are not applied to `CConfig` yet. Where the formatting is applied,t is always `function()`  the `void` argument comes from the C inheritance, not used in any modern C++ code. 

The rest of the code looks very clean, but since I don't have enough knowledge of this part, I cannot give feedback 👍🏼

Reply:
Thanks for the feedback!

Reply:
Yep it's in the plan to auto-apply clang-format to more files, we need to check if there are big PRs with changes to CConfig, and if not we can start formatting it.